be/src/exec/parquet_writer.h

```diff
@@ -57,6 +57,7 @@ class RollingAsyncParquetWriter {
 
     Status append_chunk(Chunk* chunk, RuntimeState* state);
     Status init();
+    Status cancel(RuntimeState* state);
     Status close(RuntimeState* state);
```

Comment:
we should do the cleanup job in `close`.

Reply:
done